src/include/planner/plannodes/aggregate_plan_node.h

```diff
@@ -0,0 +1,213 @@
+#pragma once
+
+#include <memory>
+#include <numeric>
+#include <string>
+#include <utility>
+#include <vector>
+#include "common/hash_util.h"
+#include "parser/expression/abstract_expression.h"
+#include "parser/expression_defs.h"
+#include "planner/plannodes/abstract_plan_node.h"
+
+// TODO(Gus, Wen): Replace Perform Binding in AggregateTerm and AggregatePlanNode
+// TODO(Gus, Wen): Replace VisitParameters
+// TODO(Gus, Wen): figure out global aggregates
+
+namespace terrier::planner {
+
+/**
+ * Plan node for aggregates
+ */
+class AggregatePlanNode : public AbstractPlanNode {
+ public:
+  /**
+   * Information for each term being aggregated on
```

Comment:
An example would help. It's not clear what this is still.

Reply:
Added an example. Let me know if this is clear